src/components/MarkerPopup/mapCustomTypeToReactComponent.jsx

```diff
@@ -16,6 +17,16 @@ export const mapCustomTypeToReactComponent = customValue => {
                     {valueToDisplay}
                 </a>
             );
+        case 'CTA':
+            return (
+                <button
+                    onClick={() => alert('Button clicked!')}
+                    style={MarkerCTAButtonStyle}
+                    variant="contained"
+                >
+                    CTA
```

Comment:
There should be "display_value" probably shown instead CTA

Reply:
Also - this button should redirect somewhere for now (later there will be form popup, but for now redirection should be enough).